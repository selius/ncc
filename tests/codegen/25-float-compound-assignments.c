int main()
{
	float a;
	a = 5.5;

	__print_float(a += 3.5);
	__print_float(a -= 4.2);
	__print_float(a *= 2.0);
	__print_float(a /= 1.2);

	return 0;
}
