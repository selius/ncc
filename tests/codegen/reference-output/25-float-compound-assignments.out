9.000000
4.800000
9.600000
8.000000
